core/configuredivecomputerthreads.cpp

```diff
@@ -2218,7 +2218,11 @@ void FirmwareUpdateThread::run()
 	}
 	switch (dc_device_get_type(m_data->device)) {
 	case DC_FAMILY_HW_OSTC3:
+#ifdef DC_FIELD_STRING
```

Comment:
What does this define do, and how does this relate to the support for forced updates on OSTC4?

Reply:
It's not related to the OSTC4 firmware upgrade at all. It's just a random macro that happens to be present in the subsurface fork of libdivecomputer, but not in the upstream version, and which is used here to distinguish between the two.

Reply:
I looked at this and it would have been good if there where a macro either identifying the subsurface fork or the force update feature but until then I'd say this is enough.

Maybe add a comment about it to clear any future confusion.